nucleus/dataset.py

```diff
@@ -1392,3 +1393,45 @@ def _upload_items(
 
         populator = DatasetItemUploader(self.id, self._client)
         return populator.upload(dataset_items, batch_size, update)
+
+    def update_scenes_metadata(self, mapping: Dict[str, dict]):
```

Comment:
Up for discussion, but we can name these `update_scenes_metadata` to `update_scenes_metadata_raw`,
to communicate that little validation is done.

Reply:
Alternatively we can swap the parameters as such:
- `update_scenes_metadata(mapping: dict)` -> `update_scenes_metadata(List[Scene])`
- `update_items_metadata(mapping: dict)` -> `update_items_metadata(List[Frame | DatasetItem])`

Then just extract the mapping internally.

Reply:
I'm applying these two changes right now actually. Commit coming in 1-2 hours.

Reply:
Haven't started yet. But to clarify. You prefer to keep this as is?

Reply:
Cool, left it as is then.